js/components/ground.js

```diff
@@ -17,20 +17,20 @@ export default class Ground extends PureComponent {
     for (let i = 0; i < MARKS_COUNT; i++) {
       const x = c.rampEndX + step * i
       const xPos = sx(x)
-      let points = [xPos, yPos, xPos, yPos + markHeight]
+      const points = [xPos, yPos, xPos, yPos + markHeight]
       lineMarks.push(<Line points={points} closed={false} stroke={'white'} strokeWidth={1} key={'mark' + x} />)
       lineMarks.push(<Text x={xPos - 9} y={yPos + markHeight * 1.2} fontFamily={'Arial'} fontSize={14} text={`${x.toFixed(1)} m`} fill={'white'} key={'text' + x} />)
     }
     return lineMarks
   }
 
   render () {
-    const { sx, sy, pixelMeterRatio } = this.props
+    const { sx, sy, pixelMeterRatio, hideMarks } = this.props
     const width = (c.runoffEndX - c.rampStartX) * pixelMeterRatio
     return (
       <Group>
         <Rect x={sx(c.rampStartX)} y={sy(c.rampBottomY)} width={width} height={GROUND_HEIGHT} fill={'green'} stroke={'black'} strokeWidth={1} />
-        { this.renderMarks() }
+        { hideMarks ? null : this.renderMarks() }
```

Comment:
```suggestion
        { !hideMarks && this.renderMarks() }
```
Six of one, half-dozen of another, but I've come to like this form as it seems simpler/more readable than the ternary/null variant (which I used to use extensively).